lib/traject/macros/general.rb

```diff
@@ -0,0 +1,12 @@
+module Macros
+  # General helpers for any traject mappings, stolen originally from DLME
+  module General
```

Comment:
I think we can get this from `traject_plus`

Reply:
Nice, I've updated accordingly